osu.ElasticIndexer/BulkIndexingDispatcher.cs

```diff
@@ -110,7 +113,7 @@ internal void Run()
         /// </summary>
         private void throttledWait()
         {
-            if (delay > 0) Task.Delay(delay * 100).Wait();
+            if (delay > 0) Task.Delay(delay * 1000).Wait();
```

Comment:
can we just remove this int count thing and use an arbitrary thread sleep? i don't foresee that ever being an issue.